apps/saleor-app-checkout/config/apl.ts

```diff
@@ -0,0 +1,23 @@
+import { FileAPL } from "@saleor/app-sdk/APL";
+import invariant from "ts-invariant";
+
+const apl = new FileAPL();
+
+export const get = async (saleorApiHost: string) => {
+  const authData = await apl.get(saleorApiHost);
+
+  invariant(authData, `No auth data found for given host: ${saleorApiHost}. Is the app installed?`);
+
+  const apiUrl = `https://${authData.domain}/graphql/`;
+
+  return {
+    apiUrl,
+    saleorApiHost,
+    domain: authData.domain,
```

Comment:
"domain" in SDK is, as a matter of fact, "host".

Reply:
https://github.com/saleor/saleor-dashboard/issues/2387
https://github.com/saleor/saleor-app-sdk/issues/87